cmd/swarmctl/task/inspect.go

```diff
@@ -40,6 +40,16 @@ func printTaskStatus(w io.Writer, t *api.Task) {
 	if t.Status.State > api.TaskStateRunning {
 		fmt.Fprintf(w, "  ExitCode\t: %d\n", ctnr.ExitCode)
 	}
+
+	if t.Status.HostPorts != nil && len(t.Status.HostPorts.Ports) > 0 {
+		ports := []string{}
+		for _, port := range t.Status.HostPorts.Ports {
+			ports = append(ports, fmt.Sprintf("0.0.0.0:%d->%d/%s",
```

Comment:
We can remove that altogether. Again `swarmctl/swarmd` are just example binaries to test swarmkit independently so we can always change in the future when we support ipv6 port publishing which we do not currently.